lntest/itest/log_error_whitelist.txt

```diff
@@ -237,6 +237,16 @@
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: rpc error: code = Canceled desc = context canceled
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: rpc error: code = DeadlineExceeded desc = context deadline exceeded
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: rpc error: code = Internal desc = transport: transport: the stream is done or WriteHeader was already called
+<time> [ERR] RPCS: [/lnrpc.Lightning/OpenChannel]: output amount(<amt>) after subtracting fees(<amt>) below dust limit(<amt>)
```

Comment:
We stopped maintaining this file and the substitutions below, it was too tedious. So feel free to remove this from the diff if you feel like it.